website/src/examples/basic-map.mdx

```diff
@@ -1,5 +1,7 @@
 # Basic Map
 
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
 import App from 'website-examples/basic-map/src/app';
 
-<App />
+<BrowserOnly>{() => <App />}</BrowserOnly>
```

Comment:
Is this the fix to get the build process to work again? I remember that at some point I thought I needed the `BrowserOnly` wrapper, but it turned out the reason for the problem was something completely different. Let's have a look if we find another way to solve this.